Neos.Neos/Classes/Service/ContentElementWrappingService.php

```diff
@@ -47,124 +46,45 @@ class ContentElementWrappingService
      */
     protected $htmlAugmenter;
 
-    /**
-     * @Flow\Inject
-     * @var NodePropertyConverterService
-     */
-    protected $nodePropertyConverterService;
-
     /**
      * Wrap the $content identified by $node with the needed markup for the backend.
      *
-     * @param NodeInterface $node
-     * @param string $content
-     * @param string $fusionPath
      * @param array $additionalAttributes additional attributes in the form ['<attribute-name>' => '<attibute-value>', ...] to be rendered in the element wrapping
-     * @return string
      */
-    public function wrapContentObject(NodeInterface $node, $content, $fusionPath, array $additionalAttributes = [])
+    public function wrapContentObject(NodeInterface $node, string $content, string $fusionPath, array $additionalAttributes = []): string
```

Comment:
we dont need this method aswell:

https://github.com/neos/neos-ui/blob/e53f8e20ee70ca832828f033f5a69c8223a2deab/Classes/Aspects/AugmentationAspect.php#L103-L112

its aop

Reply:
Thx done